Loop/Managers/DeviceDataManager.swift

```diff
@@ -364,6 +370,38 @@ final class DeviceDataManager: CarbStoreDelegate, CarbStoreSyncDelegate, DoseSto
         }
     }
 
+    /// NonMySentry Battery Calculation for Alkaline and Lithuim  #141
+    ///
+    /// - parameter currVoltage: Current Voltage Reading from Pump
+    ///
+    public var x22BatteryPercentRemaining : Double = -1
```

Comment:
Don't use magic number sentinels; use an optional. 
Since percentage is derived from chemistry, and chemistry can be changed at any time, this shouldn't be stored at all.